edifact/models/tests/segments_comparison_util.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+
+from edifact.models.interchange import InterchangeHeader, InterchangeTrailer
+from edifact.models.message import MessageHeader, MessageTrailer, NameAndAddress, DateTimePeriod, \
+    ReferenceTransactionType, ReferenceTransactionNumber
+
+
+class SegmentComparisonUtil(unittest.TestCase):
```

Comment:
this test probably doesn't run. filename needs test_ prefix

Reply:
I see its not a test, it is a helper. Maybe rename to something like "segment_assertion_helper" and a comment explaining that it is a helper for commonly used assertions?

Reply:
Also that it extends unittest.TestCase so that assertion methods can be used without needing to provide an instance of TestCase as an additional method parameter. This class is a bit confusing at first glance.

Reply:
so how to make it better? My intend was to avoid code duplication. We need that validation in two tests